src/components/MaterialUI/DataDisplay/Notification.test.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { mount } from "enzyme";
+import { Ignore } from "unexpected-reaction";
+import sinon from "sinon";
+import { Provider } from "react-redux";
+import { ignoreConsoleError } from "../../../utils/testUtils";
+import NotificationProps from "./NotificationProps";
+import Notification from "./Notification";
+import Snackbar from "@material-ui/core/Snackbar";
+import Icon from "./Icon";
+
+describe("Notification Component", () => {
+	let setSnackPack, container, store;
+	beforeEach(() => {
+		container = document.createElement("div");
+		document.body.appendChild(container);
+		setSnackPack = sinon.spy().named("setSnackPack");
+		store = {
+			subscribe: () => {},
+			dispatch: () => {},
+			getState: () => ({}),
+		};
+	});
+	afterEach(() => {
+		document.body.removeChild(container);
+		container = null;
+	});
+
+	it("Fails if notificationProps has wrong type", () => {
+		ignoreConsoleError(() => {
+			const component = <Notification notificationProps="Wrong type" />;
+			expect(() => mount(component), "to throw a", TypeError).then(error => {
+				expect(error, "to have message", "notificationProps property is not of type NotificationProps");
+			});
+		});
+	});
+
+	it("Renders Error Notification component without errors", () => {
+		const mockedMessage = {
+			key: new Date().getTime(),
+			message: "This is confirmation notification message",
+			type: "error",
+		};
+		const snackPack = [mockedMessage];
+		const autoHideDuration = 5000;
+		const anchorOrigin = { vertical: "top", horizontal: "right" };
+		const lastOnly = false;
+
+		const notificationProps = new NotificationProps();
+
+		notificationProps.set(NotificationProps.propNames.autoHideDuration, autoHideDuration);
+		notificationProps.set(NotificationProps.propNames.anchorOrigin, anchorOrigin);
+		notificationProps.set(NotificationProps.propNames.lastOnly, lastOnly);
+
+		expect(
+			<Provider store={store}>
+				<Notification notificationProps={notificationProps} snackPack={snackPack} setSnackPack={setSnackPack} />
+			</Provider>,
+			"when mounted",
+			"to satisfy",
+			<div>
+				<div>
+					<Icon id="error-cross-filled" />
+					<div>{mockedMessage.message}</div>
+					<Ignore />
```

Comment:
Don't use ignore, put what your real expectations instead. Ignore is a bad practice and should be used in rare cases.